### ISSUE TITLE
Organisation and Software Logo

### ISSUE THREAD

**Issue by el-j** [freetzing/freetzing#1]
I think we are much to close with the fritzing logo converted to blue.

i would suggested this as the new logo. as we build on javascript
![image](https://user-images.githubusercontent.com/2795534/53107659-c370d000-3535-11e9-9ea2-aff7b2af3c61.png)

**Comment by karpawich** [comment]
Yes, I assumed as much. Can you flatten the edges instead of the bezel border? And perhaps you could fill the negative space with another color like gray or sky blue so that the icon is compatible across multiple platforms - I think that it might be an issue on the desktop and mobile phones with the varying background.

**Comment by karpawich** [comment]
I can work on the changes myself, too. Suggested changes are obviously not critical.

**Comment by el-j** [comment]
is just my first suggestion, if you want you can totally change it. 
🗡

**Comment by el-j** [comment]
i think an evolving logo would be nice as well... so with every "step" we redefine and evolve the logo a bit ;)

**Comment by KingDarBoja** [comment]
I am not a fan of shadow mixed with borders, my approach would be something minimalist as the keeping the "F" logo but putting on a coloured background as the official Fritzing logo does.

You could check some existing file type logos [here](https://github.com/vscode-icons/vscode-icons/wiki/ListOfFiles) (Just as example, not because I have contribute at it) to get fresh concepts for the fritzing-js logo :) 

*Update:* Just because I am too lazy right now to think about another logo pattern (the "F" mabe by Hexagons looks a bit odd), I did some variants to the one proposed by @el-j .

![fritzing_ts_v1](https://user-images.githubusercontent.com/30560560/53121041-44719c80-3521-11e9-96c4-b3ae34270a4e.png)

![fritzing_ts_v2](https://user-images.githubusercontent.com/30560560/53121058-4f2c3180-3521-11e9-95f9-b7502241757e.png)

**Comment by el-j** [comment]
nice idea. i did a light variant:
![image](https://user-images.githubusercontent.com/2795534/53123094-89fd8c00-3558-11e9-95e1-ba9d2d53a02b.png)

**Comment by el-j** [comment]
@KingDarBoja i like the idea of the colored background. but somehow the second white line ...

**Comment by el-j** [comment]
maybe this is a better f?
![image](https://user-images.githubusercontent.com/2795534/53123642-d0071f80-3559-11e9-9986-06e876c32ca0.png)

**Comment by el-j** [comment]
@KingDarBoja the "dropshadow" is a homage to the original style of fritzing breadboard svgs

![image](https://user-images.githubusercontent.com/2795534/53124133-f083a980-355a-11e9-8f84-a8ad6c679aa0.png)

![image](https://user-images.githubusercontent.com/2795534/53124346-68ea6a80-355b-11e9-94f2-31ba6e7fa1c5.png)

**Comment by el-j** [comment]
![image](https://user-images.githubusercontent.com/2795534/53124820-a4396900-355c-11e9-8810-89baba89a668.png)

**Comment by el-j** [comment]
![image](https://user-images.githubusercontent.com/2795534/53124912-ee224f00-355c-11e9-838f-21e8fe991449.png)

**Comment by KingDarBoja** [comment]
@el-j The first "F" shape is way better that the new one you suggested few posts ago 😆 

The last one looks good to me and has some room for a "dropshadow" if you wish. By the way, notice that in my suggestions, the Hexagon was rotated 30° Degrees to keep it like a Diamond, just FYI. 

**Update:** You could try to add a slighty lighter blue to the right side of the dropshadow if you wish to have a light effect on the logo.

**Comment by el-j** [comment]
yeah, the f shape was just a try 🗡  😸 
i am not sure about the diamond rotation... i somehow like if it stands solid....
other opinions on that?

**Comment by el-j** [comment]
![image](https://user-images.githubusercontent.com/2795534/53128110-5de80800-3564-11e9-803b-a04f7b9e3c93.png)

**Comment by el-j** [comment]
close this for now. reopen if anyone has new suggestions :)

**Comment by el-j** [comment]
ok so i open this again, because i do not like the diamond style logo. 
my reasons are:
1. freetzing stands for hardware.... it is solid. the diamond is like wiggeling
2. we should adapt some of the original fritzing style, so something flat with a pseudo 2 1/2 D shadow should be used (like breadboard images have to be done)
3. the logo should for me somehow show that you can add modules. thats the little thing i have in the righty corner, as well as it seems funny like a © symbol! why not introducing this little sign as the "freetzing ©"
4. the orignal breadboard-style do not use any borders for "framing" things

> You could check some existing file type logos [here](https://github.com/vscode-icons/vscode-icons/wiki/ListOfFiles) (Just as example, not because I have contribute at it) to get fresh concepts for the fritzing-js logo :)

as far as i can see from the list node is using the diamond style. so this another argument against it. 
we base on js, so it's good to have the hexagon, but we do not do "software"... and i do not see any kind of hexagon in the list that has this orientation! so another plus for the normal style

@KingDarBoja thanks for your approach so far but i will change the logo for the organisation now. till other arguments are more convincingly!

**Comment by KingDarBoja** [comment]
@el-j Here are my new options:

<img src="https://user-images.githubusercontent.com/30560560/53291785-dfb77b80-3786-11e9-8b89-95153a764ad2.png" height="256" align="left">

<img src="https://user-images.githubusercontent.com/30560560/53291793-05dd1b80-3787-11e9-834f-cc18b3230aea.png" height="256">


I changed the fill of the "F" letter to some red-ish as legacy of the original Fritzing color, also added another hexagon to the right side as reference to "Copyright" symbol. For last, notice that the little hexagon mosaic has been merged as an path, referencing circuit copper layer.

One of them is an Octagon rather that Hexagon because was curious about the shape.

**Comment by karpawich** [comment]
For anyone who is suggesting icon designs, note that you'll eventually have to follow it with a full logo as well. Just take that into consideration when developing them 👍

**Comment by KingDarBoja** [comment]
It has its squared version for website too! :D 

<img src="https://user-images.githubusercontent.com/30560560/53292025-6ae64080-378a-11e9-900b-3d0386d2e15b.png" height="512">
